app/src/main/java/com/teamltt/carcare/DemoActivity.java

```diff
@@ -149,114 +239,53 @@ private void obdDeviceObtained() {
             socket = obdDevice.createRfcommSocketToServiceRecord(uuid);
```

Comment:
This `uuid` is never obtained from user input. We need to verify that it is the same for all devices, otherwise we need a way to discover new devices with unknown identifiers.